src/Webfactory/Slimdump/SlimdumpCommand.php

```diff
@@ -59,15 +67,47 @@ private function connect($dsn)
         }
     }
 
+    /**
+     * @return int
+     */
+    protected function parseBufferSize($size)
+    {
+        $bufferSize = $size;
+
+        if ($bufferSize !== null) {
+            $match = preg_match('/^(\d+)(KB|MB|GB)?$/', $bufferSize, $matches);
+            if ($match === false) {
+                throw new \RuntimeException('The buffer size must be an unsigned integer ending with KB, MB or GB.');
+            }
+            $bufferSize = (int)$matches[1];
+            $bufferFactor = 1;
+
+            switch ($matches[2]) {
+                case 'GB':
+                    $bufferFactor *= 1024;
+                case 'MB':
+                    $bufferFactor *= 1024;
+                case 'KB':
+                    $bufferFactor *= 1024;
+            }
+
+            return $bufferSize * $bufferFactor;
+        } else {
+            // Default 100MB
+            return 100 * 1024 * 1024;
```

Comment:
We already have the default buffer size (100 * 1024 * 1024) in `Dumper::__construct()`, so it would be great if we would not need this default case here.

Do you think we could have a default value for the command line option, or alternatively, don't call the `parseBufferSize` method when the option is not provided - so that ultimately, there's one place where the default is provided and `parseBufferSize` is only needed when there's actually something to parse?needs to